### ISSUE TITLE
The button beside session button "join and start collaboration" have no tool tip in display when keyboard focus is received to each.

### ISSUE THREAD

**Issue by fsteffi** [microsoft/vscode#178410]
[“Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags**:#A11yMAS;#A11yTCS;#Win32;#DesktopApp;#A11y_VSCode;#WCAG1.4.13;#A11ySev2;#CAITestPass_May2020;#Benchmark;#Visual Studio Code;#Keyboard;#AINotInScope;#win11;#A11ywcag2.1;

## Environment Details:
Application Name: Visual Studio Code
Version: 1.77.0-insider (user setup)
Commit: b9226e1ccc11625bcb42b55efb795e07ee533bb0
Date: 2023-03-24T18:57:27.096Z
Electron: 19.1.11
Chromium: 102.0.5005.196
Node.js: 16.14.2
V8: 10.2.154.26-electron.0
OS: Windows_NT x64 10.0.25300
Sandboxed: Yes

os version:
Edition	Windows 11 Enterprise Insider Preview
Version	22H2
Installed on	‎21-‎02-‎2023
OS build	25300.1000
Experience	Windows Feature Experience Pack 1000.25300.1000.0

## Repro Steps:

1. Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility tester after bug verification.
2.  Launch VS Code
3. . tab to live share and press enter
4.  tab to session drop down
5.  Use the tab key to navigate to the button "join and start collaboration"

## Actual:
On navigating to the buttons using tab key "join and start collaboration" there is no tooltip displayed for them, which is available on mouse hover.

### Same issue is observed in the following page:

1. no collaboration Planned Sessions
2. no collaboration Contacts
3. with collaboration Session Details
4. with collaboration Contacts
5. Audio Call-End Audio Call
6. Contacts-Change Your Own availability Status
7. Contacts-Collaborate with a recent contact
8. Contacts-Collaborate with suggested contact
9. ## Expected:
10. On the tab navigation to each of the button, the tooltip should be displayed.
11. On escape the tooltip should be dismissed.

## User Impact:
Keyboard users will not know what is the use of the button through the icon for the button.

## Attachment for Reference:
[The button beside session button join and start collaboration have no tool tip in display when keyboard focus is received to each..webm](https://user-images.githubusercontent.com/24580570/227921001-cc16c3cb-8479-40c2-aa89-471e69869173.webm)

**Comment by fsteffi** [comment]
logged in ref to bug https://github.com/Microsoft/vscode/issues/111017

**Comment by meganrogge** [comment]
created an issue with live-share for you. thanks

**Comment by fsteffi** [comment]
hi @meganrogge this bug cant be closed it will need to follow the external bug process 
can you please change the state back and share the bug created link 
thanks

**Comment by meganrogge** [comment]
@isidorn do we typically leave extension related accessibility issues open in our main repo?

**Comment by isidorn** [comment]
Closing as duplicate of https://github.com/microsoft/vscode/issues/132344

**Comment by fsteffi** [comment]
#closed;
